runtime/onert/core/src/ir/train/TrainingInfo.cc

```diff
@@ -31,7 +31,7 @@ bool TrainingInfo::isValid() const
   if (_optimizer_info.optim_code == onert::ir::train::OptimizerCode::Invalid)
     return false;
 
-  if (_optimizer_info.learning_rate == 0.0f)
+  if (_optimizer_info.learning_rate <= 0.0f)
```

Comment:
background : https://github.com/Samsung/ONE/pull/12340#discussion_r1432642877 

> (Optional)
> If learning rate is a negative value, outputs will move away from the minimum after training. In other words, outputs will move to reverse what gradient descent does, and nn model won't be trained at all.
So, it would be better to change the condition if this function is for checking valid info of `TrainingInfo`, not default values.